.travis.yml

```diff
@@ -1,27 +1,27 @@
-dist: xenial
+dist: focal
 language: node_js
 node_js:
-- 11
 - 10
-- 8
+- 12
+- 14
 before_install:
 - sudo apt-get -qq update
 - sudo apt-get install -y oathtool
 jobs:
   include:
   - stage: publish code coverage
-    node_js: '8'
+    node_js: '12'
```

Comment:
we should run tests in the lowest version of node supported

Reply:
Sure. We will fix this.

Reply:
Fixed